README.md

```diff
@@ -146,6 +146,29 @@ This project uses [EasyVCR](https://github.com/EasyPost/easyvcr-java) to record
 
 **Test Data:** The test suite has been populated with various helpful fixtures that are available for use, each completely independent from a particular user **with the exception of the USPS carrier account ID** (see [Unit Test API Keys](#unit-test-api-keys) for more information) which has a fallback value of our internal testing user's ID. Some fixtures use hard-coded dates that may need to be incremented if cassettes get re-recorded (such as reports or pickups).
 
+For some unit tests, you may need to mock certain API calls. The VCR utility in the test suite accepts an option list of `MockRequest` objects via the `setUpTest` function (see [Mocking](#mocking) below).
```

Comment:
thought: would prefer a subheading here like the others title `Mocking`

Lul now I see the mocking section below. Why not have this be under that? Seems weird to find info about one thing spread across two places